.env

```diff
@@ -8,3 +8,4 @@ GAPP_CLIENT=698562827255-boetpgjge7b3o3q7c8v58drf1seg4pep.apps.googleusercontent
 GAPP_SECRET=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
 GAPP_CALLBACK_URL= 'http://localhost:8000/auth/google/callback'
 SESSION_SECRET=findMySecretBaby
+NODE_ENV=development
```

Comment:
You should remove this before we can merge it. It can possibly brake other functionality of the code.